src/main/java/org/example/persistence/ormanager/ORManagerImpl.java

```diff
@@ -28,4 +28,22 @@ static DataSource createDataSource(String url, String user, String password, Map
         System.out.println(connection.isValid(1000));
         return dataSource;
     }
+    @Override
+    public Student save(Student student){
+        Connection connection;
+        try {
+           connection = dataSource.getConnection();
+            Statement statement = connection.createStatement();
+            statement.execute("CREATE TABLE IF NOT EXISTS STUDENTS (first_name VARCHAR(255)) ;");
+            System.out.println("table created");
+           PreparedStatement ps =  connection.prepareStatement("INSERT INTO STUDENTS (first_name) values(?)");
+           ps.setString(1, student.getFirstName());
+           int rows = ps.executeUpdate();
+            System.out.println(rows + " rows affected.");
+
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+        return student;
```

Comment:
As of this, we are returning the same object (with null Id state) that was passed in the method initially without any change.
Consider getting the auto-generated ID for this object from the database, set it, and then return the complete object.